examples/app/uploadservice-s3/src/main/java/com/example/uploadservice/s3/S3UploadTaskParameters.kt

```diff
@@ -0,0 +1,55 @@
+package com.example.uploadservice.s3
+
+import android.os.Parcelable
+import com.amazonaws.regions.Regions
+import kotlinx.android.parcel.Parcelize
+import net.gotev.uploadservice.persistence.Persistable
+import net.gotev.uploadservice.persistence.PersistableData
+import java.io.File
+
+@Parcelize
+data class S3UploadTaskParameters (
+        var uploadFilepath : String,
+        var serverSubpath: String = "",
+        var bucket_name: String,
```

Comment:
bucket_name - > bucketName. Same for other variables which are using snake case, change to camelcase.